pkg/stores/proxy/proxy_store.go

```diff
@@ -445,7 +446,15 @@ func (s *Store) Create(apiOp *types.APIRequest, schema *types.APISchema, params
 	}
 
 	gvk := attributes.GVK(schema)
-	input["apiVersion"], input["kind"] = gvk.ToAPIVersionAndKind()
+	apiVersion, kind := gvk.ToAPIVersionAndKind()
```

Comment:
This changes the behavior where before the change only the GVK was considered, and after the change the input is considered first.

Imo this _could_ be breaking the UI if they were sending the wrong apiVersion / kind before but never noticed it. That seems unlikely though.

Ideally we would have the same behavior between Update and Create. @MbolotSuse what's your thought on this?

Reply:
I agree that create and update should have the same behavior. IIUC that is part of this issue, so making them behave the same way would be in scope for this issue.

As far as the "breaking" part goes, I'm not sure about that part. I don't think that this is something that the UI should be relying on, but I also don't want to potentially break the UI if we don't need to. In this case, I think the best thing to do is address the core of the issue - update should do what create does, and that means overriding for now.

Reply:
Ok, I'll change both of the methods to override the data from input. Thank you!